components/product.tsx

```diff
@@ -0,0 +1,11 @@
+import { Layout, Link, Page, Text } from '@vercel/examples-ui'
+
+export default function Product({ product }: Object) {
```

Comment:
```suggestion
export default function Product({ product }: any) {
```





> 3 \| export default function Product({ product }: Object) {
12:15:21.221

<br class="Apple-interchange-newline">



> 3 \| export default function Product({ product }: Object) {
12:15:21.221

<br class="Apple-interchange-newline">